src/main/kotlin/com/github/gradle/android/i18n/export/XlsExporter.kt

```diff
@@ -1,65 +1,56 @@
 package com.github.gradle.android.i18n.export
 
+import com.github.gradle.android.i18n.model.*
 import org.apache.poi.xssf.usermodel.XSSFWorkbook
 import org.gradle.api.Project
 import java.io.OutputStream
 
+/**
+ * An Exporter that reads Android string resources in a Project and writes an Excel spreadsheet.
+ *
+ * This class defines how to export the resources ⇒ to and Excel workbook.
+ * The parent class [AbstractExporter] defines which resources to export.
+ */
 class XlsExporter(project: Project) : AbstractExporter(project) {
 
     override fun export(outputStream: OutputStream, defaultLocale: String) {
-
-        // Workbook creation.
-        val workbook = XSSFWorkbook()
-        val sheet = workbook.createSheet("android-i18n")
-
-        // Header row.
-        mapToRows(defaultLocale).entries.forEachIndexed { entryIndex, entryRow ->
-            val (entryKey, entryValues) = entryRow
-            val sheetRow = sheet.createRow(entryIndex)
-            val keyCell = sheetRow.createCell(0)
-            keyCell.setCellValue(entryKey)
-            entryValues.forEachIndexed { valueIndex, valueText ->
-                val valueCell = sheetRow.createCell(valueIndex + 1)
-                valueCell.setCellValue(valueText)
-            }
-        }
-
-        workbook.write(outputStream)
+        loadProjectResources(defaultLocale)
+            .toWorkbook()
+            .write(outputStream)
     }
+}
 
-    private fun mapToRows(defaultLocale: String): Map<String, List<String>> {
-
-        val rows = mutableMapOf<String, MutableList<String>>()
-
-        val loadedResources = loadProjectResources(defaultLocale)
-
-        rows["key"] = loadedResources.map { it.locale }.toMutableList()
-
-        loadedResources.forEach { stringResources ->
-
-            stringResources.strings.forEach {
-                val key = it.name ?: ""
-                val value = it.text.unescapeQuotes ?: ""
-                rows.putItem(key, value)
-            }
+private fun ProjectData.toWorkbook(): XSSFWorkbook {
+    val workbook = XSSFWorkbook()
+    modules.forEach { moduleData ->
+        val sheet = workbook.createSheet(moduleData.name)
+
+        // Header row: "key" | lang1 | lang2
+        val headerRow = sheet.createRow(0)
+        headerRow.createCell(0).setCellValue("key")
+        moduleData.translations.forEachIndexed { translationDataIndex, translationData ->
```

Comment:
I would better understand the code if we replace `translationDataIndex` by `langIndex`.
(same for row 32 / 48 / 49)

WDYT?